recipe_batches/recipe_batches.py

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/python
-
 import math
 
-def recipe_batches(recipe, ingredients):
-  pass 
 
+def recipe_batches(recipe, ingredients):
+    batches_list = []
+    for recipe_ingredient in recipe:
+        try:
+            possible_batches = (ingredients[recipe_ingredient]) // (recipe[recipe_ingredient])
```

Comment:
instead of using a list and sorting it ( o(n) ), you can do something like...
```
batches = 0
for ...:
    try:
        possible_batches = (ingredients[recipe_ingredient]) // (recipe[recipe_ingredient])
        if batches == 0 or possible_batches < batches:
            batches = possible_batches
    except:
        return 0   # no batches,  missing ingredient
    return batches
````